### ISSUE TITLE
Yokosuka naval yard gone

### ISSUE THREAD

**Issue by dirtbaghippy** [1010todd/Mega-Freight#9]
I finished the fw campaign and am working through the wanderers campaign, I was looking for the Yokosuka naval yard in the Diphda system since I remember visiting there early in my save. But for some reason its no longer there, was curious if that was meant to happen or if I am experiencing a bug.

**Comment by dirtbaghippy** [comment]
I tried looking over the text files to try and make sense of how all the spawning conditions and all that stuff works but I couldnt figure it all out. I guess I am curious whether the naval yard shows up later after I do some stuff, or if it was supposed to be there without me doing anything and if it disappears for a reason. It does say in my save file:

"visited planet" "Yokosuka Naval Arsenal"

However when I search further in the save file, there is no other mention of it.

**Comment by dirtbaghippy** [comment]
After awhile of testing, it occurred to me that maybe there is plugin incompatibility since I couldn't find anything in the files about the station disappearing due to any event. I also specifically remember landing on the station at some point which made this whole thing confusing. 
Sure enough I had a plugin conflict, I had added a few plugins as I went since I started and in case anyone else just so happens to have this same issue, the "animated-stars-nova" plugin was the culprit.
